contrib/legal/privacy.html.sample

```diff
@@ -0,0 +1,194 @@
+   <li>We use your email address to communicate with you, if you've said that's okay, and only for the reasons you’ve said that’s okay.</li>
+   <li>We use User Personal Information and other data to make recommendations for you, such as to suggest projects you may want to follow or contribute to. We learn from your public behavior on Your Gitea Instance—such as the projects you star—to determine your coding interests, and we recommend similar projects. These recommendations are automated decisions, but they have no legal impact on your rights.</li>
+   <li>We use Usage Information and Device Information to better understand how our Users use Your Gitea Instance and to improve our Website and Service.</li>
+   <li>We may use your User Personal Information if it is necessary for security purposes or to investigate possible fraud or attempts to harm Your Gitea Instance or our Users.</li>
+   <li>We may use your User Personal Information to comply with our legal obligations, protect our intellectual property, and enforce our Terms of Service.</li>
+
+   <li>We limit our use of your User Personal Information to the purposes listed in this Privacy Statement. If we need to use your User Personal Information for other purposes, we will ask your permission first. You can always see what information we have, how we're using it, and what permissions you have given us in your user profile.</li>
+  </ol>
+
+  <h2>How Your Gitea Instance Secures Your Information?</h2>
+
+  <p>Your Gitea Instance takes all measures reasonably necessary to protect User Personal Information from unauthorized access, alteration, or destruction; maintain data accuracy; and help ensure the appropriate use of User Personal Information.</p>
+
+  <p>To the extent above, we enforce a written security information program, which:</p>
+
+  <ul>
+    <li>aligns with industry recognized frameworks;</li>
+    <li>includes security safeguards reasonably designed to protect the confidentiality, integrity, availability, and resilience of our Users' data;</li>
+    <li>is appropriate to the nature, size, and complexity of Your Gitea Instance’s business operations;</li>
+    <li>includes incident response and data breach notification processes; and</li>
+    <li>complies with applicable information security-related laws and regulations in the geographic regions where Your Gitea Instance does business.</li>
+  </ul>
+
+  <p>In the event of a data breach that affects your User Personal Information, we will act promptly to mitigate the impact of a breach and notify any affected Users without undue delay.</p>
+
+  <p>Transmission of data on Your Gitea Instance is encrypted using SSH, HTTPS (TLS), and git repository content is encrypted at rest. We manage our own cages and racks at top-tier data centers with high level of physical and network security, and when data is stored with a third-party storage provider, it is encrypted.</p>
```

Comment:
This is probably overkill and I doubt most instances will be doing it. It's fine to keep it in but it may wrongly give the impression that you can't run a Gitea legally without doing this. 

You probably couldn't store very sensitive personal information without doing this and so it makes sense to advertise if your Gitea is suitable for that but I doubt many services can say they control their own racks and cages.

Reply:
@zeripath Data breach mitigation overkill for small and medium deployments?

Reply:
Running a Gitea instance on aws or vps is excluded by these terms - it's not just basic data breach mitigation.

This level of protection is appropriate for medical data. It may not be suitable for a fork of Kodi protected by a 10 character password. Similarly encrypting the storage of publicly available source code which is verifiable with signatures may not be necessary - particularly as the most likely source of a leak is not going to be a data store attack but rather a security fault in Gitea, or the stack running Gitea - in which case that the filesystem is encrypted is of no consequence.

A service should advertise the level of security they are engaging in. At what point you are negligent under the GDPR has not been formally tested but I doubt it extends to owning your own server racks and cages.

It's fine to keep it in as ideal but we should remember that most users aren't running Gitea as a GitHub competitor. The average Gitea user runs this a private host for themselves plus/minus a few others. They still need to comply with the GDPR and we want to make it easy for them to do so.

GitHub competitors can afford their own bloody lawyers. We need sensible defaults.

Reply:
@zeripath so what sensible defaults then?